### ISSUE TITLE
tag version 2.0 have wrong version information in setup.cfg

### ISSUE THREAD

**Issue by hsiaoyi0504** [deepchem/deepchem#1351]
https://github.com/deepchem/deepchem/blob/2.0.0/setup.cfg#L8

This line should be 2.0.0

I get following error message when installed through `python setup.py install`:

```
ERROR:root:Error parsing
Traceback (most recent call last):
  File "/headless/miniconda/lib/python3.6/site-packages/pbr/core.py", line 111, in pbr
    attrs = util.cfg_to_args(path, dist.script_args)
  File "/headless/miniconda/lib/python3.6/site-packages/pbr/util.py", line 249, in cfg_to_args
    pbr.hooks.setup_hook(config)
  File "/headless/miniconda/lib/python3.6/site-packages/pbr/hooks/__init__.py", line 25, in setup_hook
    metadata_config.run()
  File "/headless/miniconda/lib/python3.6/site-packages/pbr/hooks/base.py", line 27, in run
    self.hook()
  File "/headless/miniconda/lib/python3.6/site-packages/pbr/hooks/metadata.py", line 26, in hook
    self.config['name'], self.config.get('version', None))
  File "/headless/miniconda/lib/python3.6/site-packages/pbr/packaging.py", line 755, in get_version
    name=package_name))
Exception: Versioning for this project requires either an sdist tarball, or access to an upstream git repository. It's also possible that there is a mismatch between the package name in setup.cfg and the argument given to pbr.version.VersionInfo. Project name deepchem was given, but was not able to be found.
error in setup command: Error parsing /headless/deepchem-2.0.0/setup.cfg: Exception: Versioning for this project requires either an sdist tarball, or access to an upstream git repository. It's also possible that there is a mismatch between the package name in setup.cfg and the argument given to pbr.version.VersionInfo. Project name deepchem was given, but was not able to be found.
```

**Comment by hsiaoyi0504** [comment]
It seems that I install deepchem into wrong python version, but it would be good to update the setup.cfg as well.

**Comment by hsiaoyi0504** [comment]
It turns out that you may need to `export PBR_VERSION=2.0.0` during installation. @rbharath Can we have a new tagged version for 2.0.0 ? or like 2.0.1 with the fix of the version ?

Another thing is that I can open a pull request to include the information I mentioned above.

**Comment by rbharath** [comment]
I think this old issue is no longer relevant (since we're now on version 2.3.0 released and 2.3.1 in the `setup.cfg` for HEAD). I'm going to go ahead and close.
